src/library/Uno.Material/Converters/FromOpenToCustomValueConverter.cs

```diff
@@ -8,7 +8,7 @@
 using Windows.UI.Xaml.Data;
 #endif
 
-namespace Uno.Material.Converters
+namespace Uno.Material
 {
 	public class FromOpenToCustomValueConverter : IValueConverter
```

Comment:
i know you didnt write this, but we should drop this class and replace it by FromBoolToValueConverter

Reply:
ok will look into it, thank you! But I think it will be better in a separated PR ;)